flang/test/Driver/dynamic-linker.f90

```diff
@@ -7,13 +7,17 @@
 ! RUN: %flang -### --target=x86_64-windows-msvc -rpath /path/to/dir -shared \
 ! RUN:     -static %s 2>&1 | FileCheck \
 ! RUN:     --check-prefixes=MSVC-LINKER-OPTIONS %s
+! RUN: %flang -### --target=aarch64-linux-none -rdynamic %s 2>&1 | FileCheck --check-prefixes=AARCH-LINKER-OPTIONS %s
```

Comment:
[nit] This run line is specifically for `-rdynamic` and the prefix should reflect that. Perhaps `RDYNAMIC`?

Reply:
Sounds reasonable, will update soon. Thanks!